sw/device/tock/dif/build.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+extern crate bindgen;
+
+use bindgen::builder;
+use std::env;
+use std::fs::{self};
+use std::path::Path;
+
+fn main() {
+    let source_root = env::var("MESON_SOURCE_ROOT").unwrap();
+    let build_root = env::var("MESON_BUILD_ROOT").unwrap();
+    let out_dir = env::var("OUT_DIR").unwrap();
+
+    generate_bindings(&source_root, &build_root, &out_dir);
+
+    let dif_lib_path = Path::new(&build_root).join("sw/device/lib/dif");
+    let dif_libs = &["uart_ot"];
```

Comment:
I would consider inlining this variable into the call below. Creating a variable for a single use like this isn't very common in Rust.

Reply:
Done.